tests/fast-integration/telemetry-test/suite.js

```diff
@@ -458,6 +481,55 @@ describe('Telemetry Operator', function() {
           const secret = await getSecret('telemetry-trace-collector', 'kyma-system');
           assert.equal(fromBase64(secret.data.OTLP_ENDPOINT), 'http://another-foo-bar');
         });
+
+        it(`Should delete TracePipeline`, async function() {
+          await k8sDelete(pipeline);
+        });
+      });
+
+      context('Filter Processor', function() {
+        const testApp = loadTestData('tracepipeline-test-app.yaml');
+        const testAppIstioPatch = loadTestData('tracepipeline-test-istio-telemetry-patch.yaml');
+        const testAppIstioCleanup = loadTestData('tracepipeline-test-istio-telemetry-cleanup.yaml');
+
+        it(`Should create test app`, async function() {
+          const kymaGateway = await getGateway('kyma-system', 'kyma-gateway');
+          let kymaHostUrl = kymaGateway.spec.servers[0].hosts[0];
+          kymaHostUrl = kymaHostUrl.replace('*', 'tracing-test-app');
+          for (const resource of testApp ) {
+            if (resource.kind == 'VirtualService') {
+              resource.spec.hosts[0] = kymaHostUrl;
+            }
+          }
+          await k8sApply(testApp);
+          await k8sApply(testAppIstioPatch);
+          await waitForPodWithLabel('app', 'tracing-test-app', 'tracing-test');
+        });
+
+        it(`Should call test app and produce spans`, async function() {
+          await sleep(30000);
+          for (let i=0; i < 10; i++) {
+            await callTracingTestApp();
+          }
+        });
+
+        it(`Should filter out noisy spans`, async function() {
+          await sleep(30000);
+          const services = await getJaegerServices();
+          const testAppTraces = await getJaegerTracesForService('tracing-test-app', 'tracing-test');
+          assert.isTrue(testAppTraces.data.length > 0, 'No spans present for test application "tracing-test-app"');
+
+          assert.isFalse(services.data.includes('grafana.kyma-system'), 'spans are present for grafana');
+          assert.isFalse(services.data.includes('jaeger.kyma-system'), 'spans are present for jaeger');
+          assert.isFalse(services.data.includes('telemetry-fluent-bit.kyma-system'),
+              'spans are present for fluent-bit');
+          assert.isFalse(services.data.includes('loki.kyma-system'), 'spans are present for loki');
+        });
+
+        it(`Should delete test setup`, async function() {
+          await k8sApply(testAppIstioCleanup);
```

Comment:
can we reuse the variable:` testAppIstioPatch` and update the value. To keep the number of files in testdata under control.

Reply:
istio cleanup resource removed